src/components/TimeSlider.js

```diff
@@ -500,6 +509,7 @@ export default class CountryInfo extends Component {
       firstDay: '',
       lastDay: '',
       currentSliderRange: [],
+      CurrentRange: mobileRange,
```

Comment:
Should we keep using camelCase here?

```suggestion
      currentRange: mobileRange,
```